source/octf/trace/parser/TraceEventHandler.h

```diff
@@ -97,6 +97,8 @@ class TraceEventHandler {
         return true;
     }
 
+    typedef EventType eventType;
```

Comment:
Suggestion: HandlerEventType

Reply:
I prefer to use the shorter version, I think it's still clear what kind of event it is.